HunterPie.Core/Game/Enums/CombatStatus.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace HunterPie.Core.Game.Enums;
+
+public enum CombatStatus
+{
+    None,
+    Battle,
+    Caution,
+    Battle_Move
```

Comment:
```suggestion
    BattleMove
```
What does `BattleMove` even mean? Also, keep enums value names in C# code standard, please.

Reply:
`EnemyMove` looks more suitable. I'll fix it!

Reply:
Oh, it's when the monster is running away?

Reply:
Yes :)